crates/viewer/re_recording_panel/src/recording_panel_ui.rs

```diff
@@ -590,13 +595,31 @@ fn receiver_ui(
             }
         });
 
-    if show_hierarchal {
+    let response = if show_hierarchal {
         ui.list_item()
             .selected(selected)
-            .show_hierarchical(ui, label_content);
+            .show_hierarchical(ui, label_content)
     } else {
         ui.list_item()
             .selected(selected)
-            .show_flat(ui, label_content);
-    }
+            .show_flat(ui, label_content)
+    };
+
+    response.context_menu(|ui| {
+        let url = ViewerOpenUrl::from_data_source(receiver).and_then(|url| url.sharable_url(None));
+        if ui
+            .add_enabled(url.is_ok(), egui::Button::new("Copy link to partition"))
+            .on_disabled_hover_text("Can't copy a link to this partition")
```

Comment:
Should we print a reason? 🤔 
I forget whether that was a well printable error or not